packages/framework/assets/js/admin/components/CkEditorPreview.js

```diff
@@ -0,0 +1,45 @@
+import Register from '../../common/utils/Register';
+import Translator from 'bazinga-translator';
+
+export default class CkEditorPreview {
+
+    constructor ($ckEditorPreview) {
+        const expandText = Translator.trans('... expand ...');
```

Comment:
```suggestion
        const expandText = Translator.trans('Click here to edit this text');
```

and use this for all CKEditors not only with longer texts?

Reply:
I'm not sure about the best approach. I would like to wait for the UX guy's opinion as we planned.

Reply:
Reworked, so now only edit button is present (expand preview text functionality was removed)